libcxx/cmake/caches/hexagon-linux-runtimes.cmake

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Meta comment: As part of our support policy, we would like to have a CI job that builds and tests libc++ for Hexagon. I'm not very familiar with that target, but do you think you could set that up?

Reply:
IMO landing these cache files are baby steps towards having such a CI job.  I'll look into what it would take, yes.

Reply:
Ack. Just be aware that so long as there's no CI job, the platform is not officially supported so it may happen that we'll make changes that will make your life harder. We won't do it on purpose, of course, but it could happen because "no CI" == "no visibility into the impact of changes for that configuration".

Reply:
Absolutely.  That's been the status quo and this PR on its own can't change that.

The intent of this pull req is to allow anyone to build a toolchain for targeting hexagon (of which libc++, libc++abi, libunwind are a part).  But another benefit can be that we should be able to setup CI job(s) leveraging this, indeed.  We can enable regression tests for the entire toolchain including compiler, linker *and* target libraries.